cfpb/capital-framework
# Basic clean up of icons before atomic work

Super small cleanup of icons before I start working with them. Started with the variables because I wanted to be sure I don't break anything in the process.
## Additions
- Added icons to link script
## Changes
- Updated unicode entries with variables
- Updated spacing to be consistent
## Testing
- Nothing really changed, but if you want to be sure the variables are correct, run `npm run cf-link` in this repo
- Open the sandbox and run `npm link cf-icons && npm run build && npm start`
- Open `http://localhost:3000/components/cf-icons/` and ensure the icons are all what they should be
## Review
- @Scotchester 
- @KimberlyMunoz 
## Screenshots
## Notes
- This is only a pre-emptive change and while it may not be necessary once we move to svg, it'll be helpful for the time being.
## Checklist
